packages/kbn-interpreter/.babelrc

```diff
@@ -1,6 +1,7 @@
 {
   "presets": ["@kbn/babel-preset/webpack_preset"],
   "plugins": [
+    "@babel/plugin-transform-modules-commonjs",
```

Comment:
This project is required from both server/client code and only has a single build output - a follow-up could be to build separate packages but out of the scope for this PR.

Reply:
Hmm, yeah, it used to build separate outputs...

Reply:
The plan is to [remove this package eventually](https://github.com/elastic/kibana/issues/57340) as the `expressions` plugin should already be exposing all of this functionality.